packages/create-redwood-app/template/.env.defaults

```diff
@@ -12,7 +12,11 @@ DATABASE_URL=file:./dev.db
 # disables Prisma CLI update notifier
 PRISMA_HIDE_UPDATE_MESSAGE=true
 
+
 # Option to override the current environment's default api-side log level
 # See: https://redwoodjs.com/docs/logger for level options:
 # trace | info | debug | warn | error | silent
 # LOG_LEVEL=debug
+
+WEBHOOK_SECRET=THIS_IS_NOT_SECRET
```

Comment:
Should we leave some docs about why this is here, and should it be uncommented by default?

Reply:
Yes, good point. Will do.